### PR TITLE
Add disabled concept pill to demo

### PR DESCRIPTION
**Pull request by laurieboyes**
🐿 v2.5.13

### CODE REVIEW

File: demos/templates/demo.html [hunk 1]
```diff
@@ -58,6 +58,19 @@
 		</div>
 	</div>
 
+<div class="o-grid-row">
+		<div data-o-grid-colspan="12">
+			<h2>Concept pill disabled</h2>
+
+			<ul class="concept-pill-collection">
+				<li>
+					<span class="n-myft-concept-pill n-myft-concept-pill--disabled" href="#">Books</span>
```
Comment (keirog) [inline demos/templates/demo.html]: ~`span`~ `a`?

File: demos/templates/demo.html [hunk 2]
```diff
@@ -58,6 +58,19 @@
 		</div>
 	</div>
 
+<div class="o-grid-row">
```
Comment (keirog) [inline demos/templates/demo.html]: Tab

File: demos/templates/demo.html (hunk 1 above)
Comment (GlynnPhillips) [inline demos/templates/demo.html]: By the looks of it this is the non-clickable version so `span` is correct ;)

Comment (laurieboyes) [inline demos/templates/demo.html]: Oh except I left `href` in by accident, whoops 🙃

**Pull request merged**